test/src/e2e_vm_tests/test_programs/chained_if_let/Forc.toml

```diff
@@ -0,0 +1,5 @@
+[project]
+authors = ["Alexander Hansen"]
```

Comment:
```suggestion
authors = ["Fuel Labs <contact@fuel.sh>"]
```

Reply:
How much do we care about this?  Most of the tests are `Fuel Labs`, I think @adlerjohn changed them all initially.

Reply:
They should be Fuel Labs for consistency at the very least.